### ISSUE TITLE
#error in Miscellaneous files results in squiggles and error in Error List window

### ISSUE THREAD

**Issue by jeremymeng** [dotnet/roslyn#22386]
**Version Used**: VS 2017

**Steps to Reproduce**:

1. open a C# file outside of a project that builds fine so it is put in Miscellaneous Files project
2. past the following code into the file
```csharp
public class P
{
#error this is an error

    Unknown T;
}
```

**Expected Behavior**:
No squiggles for the 'error' line, as we don't show errors for other lines with error (e.g., `Unknown T`)

**Actual Behavior**:
Squiggle for 'error' line, and an error for it in Error List window

**Comment by Pilchie** [comment]
Miscellaneous files *do* show syntax errors, just not semantic ones.

**Comment by jeremymeng** [comment]
Ah that make sense.  Thanks @Pilchie!  Closing.
